bin/custom/bookstore/bookstorecore/src/my/bookstore/core/jobs/BookJobPerformable.java

```diff
@@ -0,0 +1,149 @@
+/**
+ *
+ */
+package my.bookstore.core.jobs;
+
+import de.hybris.platform.cronjob.enums.CronJobResult;
+import de.hybris.platform.cronjob.enums.CronJobStatus;
+import de.hybris.platform.cronjob.model.CronJobModel;
+import de.hybris.platform.servicelayer.cronjob.AbstractJobPerformable;
+import de.hybris.platform.servicelayer.cronjob.PerformResult;
+import de.hybris.platform.servicelayer.model.ModelService;
+
+import java.util.List;
+
+import org.apache.log4j.Logger;
+
+import my.bookstore.core.model.BookModel;
+import my.bookstore.core.services.RentalService;
+
+
+/**
+ * @author ecapozza
+ *
+ */
+public class BookJobPerformable extends AbstractJobPerformable<CronJobModel>
+{
+	private static final Logger LOG = Logger.getLogger(BookJobPerformable.class);
+	RentalService rentalService;
+	ModelService modelService;
+
+	/**
+	 * @return the modelService
+	 */
+	public ModelService getModelService()
+	{
+		return modelService;
+	}
+
+	/**
+	 * @param modelService
+	 *           the modelService to set
+	 */
+	@Override
```

Comment:
perchè mettere l'Override sul set?

per rendere la classe leggibile, in genere i metodi get e set vanno messi sotto a tutti gli altri metodi, in genere l'ordine è questo:

* metodo costruttore (se serve)
* metodi public
* metodi private
* metodi get e set